tvbwidgets/ui/ts_widget.py

```diff
@@ -203,11 +262,12 @@ def __init__(self, **kwargs):
         self.sample_freq = 0
 
         self.output = widgets.Output(layout=widgets.Layout(width='auto'))
+        self.annotation_area = self._create_annotation_area()
```

Comment:
`self.annotation_area`  does not need to be an attribute on the class. I think it can safely be a variable, as it is only used in this context.